crypto/evp/evp_local.h

```diff
@@ -122,6 +122,8 @@ struct evp_keymgmt_st {
     OSSL_FUNC_keymgmt_gen_fn *gen;
     OSSL_FUNC_keymgmt_gen_cleanup_fn *gen_cleanup;
 
+    OSSL_FUNC_keymgmt_load_fn *load;
```

Comment:
If you are going to use this name, should we also be using the name save?

Reply:
That would be a future extension.